platformio.ini

```diff
@@ -8,7 +8,30 @@
 ; Please visit documentation for the other options and examples
 ; https://docs.platformio.org/page/projectconf.html
 
-[env:disco_f072rb]
+[env]
 platform = ststm32
 board = disco_f072rb
 framework = zephyr
+test_framework = unity
+lib_deps =
+    lab
+
+[env:disco_board]
+monitor_speed = 115200
+
+[env:disco_renode]
+platform_packages =
+    platformio/tool-renode
+test_testing_command =
+    ${platformio.packages_dir}/tool-renode/renode
+    --disable-xwt
+    --console
```

Comment:
I like the way you have divided up the environment for the testing and uploading.